### PR TITLE
Imran branch2

### LINKED ISSUE #242: lack of images
* **Describe the bug**

want to see images in table

1. check code
2. check missing image 

* **Expected behavior**

image in table

* **Platform**

Were you running the code on:

- [x] Local R
- [ ] Citrix
- [ ] Non-network laptop

* **Screenshots**

### PR DESCRIPTION
**Pull request by Dipad-imrankhan**
## Proposed changes

Describe the big picture of your changes here to make it clear what your pull request will change. Make sure to explain the **what** e.g. the functionality it changes not the **how** e.g. what code you have written.  If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [x] Bugfix (change which fixes an issue)
- [x] New feature (change which adds functionality)
- [ ] Documentation Update (change to naming or other documentation)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR._

- [ ] I have added unit tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [x] I have checked that my changes have not broken any other functionality
- [ ] I have linked to any issues this PR fixes 

## Points for review

Add checkboxes here for any specific aspects of your PR you would like to be checked in peer review. If you don't specify anything here, the reviewer will check:

- [ ] All of your unit tests pass
- [x] Your code is clear and easy to understand
- [ ] You have documented any new features
- [ ] They can run the code you have written
- [x] Your changes do not break any other functionality

## Who is reviewing this PR?

Tag your reviewer here to ensure they get a notification!

### CODE REVIEW

File: text_file.txt [hunk 1]
```diff
@@ -3,3 +3,7 @@ This is a random text file. You can add and remove things to it to test how git
 
 This is some text.
 I am just normally editing a text file here.
+
+my name is imran
+
+hi I'm Maggie.
```
Comment (avasharma) [inline text_file.txt:9]: good addition to this code

File: text_file.txt [hunk 2]
```diff
@@ -3,3 +3,7 @@ This is a random text file. You can add and remove things to it to test how git
 
 This is some text.
 I am just normally editing a text file here.
+
+my name is imran
+
```
Comment (avasharma) [inline text_file.txt:8]: great addition to this code

File: text_file.txt [hunk 3]
```diff
@@ -3,3 +3,7 @@ This is a random text file. You can add and remove things to it to test how git
 
 This is some text.
 I am just normally editing a text file here.
+
+my name is imran
```
Comment (AlexWilliamsDfT) [inline text_file.txt:7]: great name

File: text_file.txt (hunk 1 above)
Comment (AlexWilliamsDfT) [inline text_file.txt:9]: another great name

**Pull request closed without merging**